src/main/java/net/gecore/streamerendpoints/controller/IGDBController.java

```diff
@@ -0,0 +1,41 @@
+package net.gecore.streamerendpoints.controller;
+
+import java.util.List;
+
+import net.gecore.streamerendpoints.domain.TwitchGame;
+import net.gecore.streamerendpoints.service.igdb.IGDBAPIException;
+import net.gecore.streamerendpoints.service.igdb.IGDBGameService;
+import net.gecore.streamerendpoints.service.twitch.StreamService;
+import net.gecore.streamerendpoints.service.twitch.TwitchAPIException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@RestController
+@RequestMapping("${rest.api}" + "/user")
+public class IGDBController {
+
+    private final IGDBGameService igdbGameService;
+    private final StreamService streamService;
+
+    @Autowired
+    public IGDBController(IGDBGameService igdbGameService, StreamService streamService){
+        this.igdbGameService = igdbGameService;
+        this.streamService = streamService;
+    }
+
+    @GetMapping(value = "/{userId}/stream/game")
+    public String retrieveGameInfo(@PathVariable Integer userId,
+        @RequestParam(required = false) List<String> filters)
```

Comment:
Remove the fields thing for now, it's not used yet. It will be implemented later. For now it'll be forced to the url and summary return.